deploy/osd-pcap-collector/pcap-collector-scc.yaml

```diff
@@ -0,0 +1,20 @@
+kind: SecurityContextConstraints
+apiVersion: v1
+metadata:
+  name: pcap
+allowPrivilegedContainer: true 
+allowHostNetwork: true
+allowedCapabilities:
+- 'NET_ADMIN'
+- 'NET_RAW'
+runAsUser:
+  type: MustRunAsNonRoot
+seLinuxContext:
+  type: RunAsAny
+users:
+- system:serviceaccount:sbr-hosted-pcap:default
+groups:
```

Comment:
Why are we assigning this to groups in this way? This will make this the default SCC for these groups.